### ISSUE TITLE
Parse attribute descriptions for custom types

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-tasks-core#400]
With #383, we are now able to parse the description of first-level arguments (the arguments of a task function), and include them into the schema. After #386, it will be useful to also parse the description of attributes of first-level arguments, when these are some custom models we defined (e.g. the description of `Channel.label`).

Ref: https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/386#issuecomment-1580764017

**Comment by tcompa** [comment]
Closing in favor of https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/418
